ra/ra_cfg/fsp_cfg/bsp/ra8m1/bsp_mcu_device_pn_cfg.h

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2024 Renesas Electronics Corporation
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef BSP_MCU_DEVICE_PN_CFG_H_
+
+#if defined(CONFIG_SOC_R7FA8M1AHECBD)
+#define BSP_MCU_R7FA8M1AHECBD
+#define BSP_MCU_FEATURE_SET       ('A')
+#define BSP_ROM_SIZE_BYTES        (2064384)
+#define BSP_RAM_SIZE_BYTES        (917504)
+#define BSP_DATA_FLASH_SIZE_BYTES (12288)
```

Comment:
We could pull the values from DT.

Reply:
@ydamigos : These values are dummy now. This file is necessary building with fsp only. We would like to keep it in this PR.

Reply:
@thaoluonguw We could keep the file but please remove unused macros. We use only `BSP_PACKAGE_PINS`. Let's just leave this one only.

Reply:
Now, We can use `substring` function in Kconfig.
Some definitions can be calculate from P/N.

Reply:
@ydamigos : We updated it.